performant-software/neatline-omeka-s
# Add API operations for creating and retrieving Neatline records

### What does this PR do?
- Adds API resources for Neatline records
- Installs a Doctrine ORM extension for handling spatial data types and importing/exporting GeoJSON
- Moves the Neatline view container into the Omeka site route

### What issues does it address?
- Closes #2 

### How to test
- Visit https://neatline-3-staging.herokuapp.com/s/public-site/neatline to load the top-level Neatline view for a public Omeka S site
- Click "Public View" under the first exhibit in the table. This should load a map.
- The map should show one or more records — any that have been created and assigned to this exhibit via the API — as markers on the map.
